src/calendarmanager.cpp

```diff
@@ -394,12 +394,17 @@ void CalendarManager::updateAgendaModel(CalendarAgendaModel *model)
                 continue;
             }
 
-            const QDateTime startDt(model->startDate()); // To be replaced later by start.startOfDay()
-            const QDateTime endDt(model->endDate()); // To be replaced later by start.startOfDay()
+#if (QT_VERSION >= QT_VERSION_CHECK(5, 14, 0))
+            const QDateTime startDt(model->startDate().startOfDay());
+            const QDateTime endDt(model->endDate().endOfDay());
+#else
+            const QDateTime startDt(model->startDate());
+            const QDateTime endDt(model->endDate()).addDays(1);
```

Comment:
Ah, actually, this doesn't compile, it should be written like `const QDateTime endDt(QDateTime(model->endDate()).addDays(1));`

Reply:
I was confused since I compiled and tested it, but then I realized I was only testing the parts with the newer version of Qt.  My apologies!

Reply:
No problem.